cilium/cmd/bpf_egress_list.go

```diff
@@ -15,20 +17,48 @@ import (
 )
 
 const (
-	egressListUsage = "List egress entries.\n" + lpmWarningMessage
+	egressListUsage = "List egress policy entries.\n" + lpmWarningMessage
 )
 
+type EgressPolicy struct {
+	SourceIP string
+	DestCIDR string
+	EgressIP string
+	Gateways []string
+}
+
 var bpfEgressListCmd = &cobra.Command{
 	Use:     "list",
 	Aliases: []string{"ls"},
-	Short:   "List egress entries",
+	Short:   "List egress policy entries",
 	Long:    egressListUsage,
 	Run: func(cmd *cobra.Command, args []string) {
 		common.RequireRootPrivilege("cilium bpf egress list")
 
-		bpfEgressList := make(map[string][]string)
-		if err := egressmap.EgressMap.Dump(bpfEgressList); err != nil {
-			Fatalf("error dumping contents of map: %s\n", err)
+		egressmap.InitEgressMaps()
```

Comment:
IIUC, calling this command when the egress gateway is disabled will cause us to create a dummy map because `InitEgressMaps` ends up calling `OpenOrCreate`. Maybe we should avoid that? I don't think the other `cilium bpf` commands have this side effect.

Reply:
Yep makes sense, I'll add a new method to just open the existing pinned map and switch these commands to it :+1: